src/sqs/mod.rs

```diff
@@ -80,29 +81,25 @@ impl SyncEventQueue {
     }
 
     /// Acknowledge a received SQS message by deleting it from the queue.
-    pub fn ack_message(&self, event: &SyncEvent) -> Result<()> {
+    pub async fn ack_message(&self, event: &SyncEvent) -> Result<()> {
         let sqs = self.sqs.clone();
         let mut request = DeleteMessageRequest::default();
         request.queue_url = self.url.clone();
         request.receipt_handle = event.handle.clone();
         debug!(self.logger.log, "SQS Acking message {:?}", request);
         sqs.delete_message(request)
-            .with_timeout(Duration::new(1, 0))
-            .sync()?;
+            .await
```

Comment:
Do we want to keep the timeouts? If so I think we can to wrap these futures with `tokio::time::timeout` which should be the equivalent?